app/presenters/queries/expanded_link_set.rb

```diff
@@ -5,7 +5,7 @@ def initialize(content_id:, state_fallback_order:, locale_fallback_order: Conten
         @content_id = content_id
         @state_fallback_order = Array(state_fallback_order)
         @locale_fallback_order = Array(locale_fallback_order)
-        @visited = []
+        @level_index = {}
```

Comment:
This variable confused me a bit - as I understand it's used just in the recursive process on dependees and would cause problems if there were more multiple recursive processes. I wonder if it makes more sense being injected into children from the dependees method rather than being an instance variable?